pkg/server/login/login.go

```diff
@@ -169,20 +170,25 @@ func (l *Login) handleLogin(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	audit.AddUsername(req, username)
+
 	authResponse, ok, err := l.auth.AuthenticatePassword(context.TODO(), username, password)
 	if err != nil {
 		utilruntime.HandleError(fmt.Errorf(`Error authenticating %q with provider %q: %v`, username, l.provider, err))
 		failed(errorpage.AuthenticationErrorCode(err), w, req)
+		audit.AddDecision(req, audit.ErrorDecision)
```

Comment:
note to self: both `AuthenticatorRequestWithAuditDecision(authReq authenticator.Request)` and this is adding decisions, that's rather odd

Reply:
I think that handleLogin doesn't go the path of authenticate.Request.

Reply:
no follow up, so I assume it is resoled.